conformance/utils/http/http.go

```diff
@@ -44,9 +45,15 @@ type ExpectedRequest struct {
 	Headers map[string]string
 }
 
-// MakeRequestAndExpectResponse makes a request with the given parameters and
-// verifies the response matches the provided ExpectedResponse.
-func MakeRequestAndExpectResponse(t *testing.T, r roundtripper.RoundTripper, gwAddr string, expected ExpectedResponse) {
+const maxConsistencyPeriodPerRequest = 60 * time.Second
+const numConsistencyChecksPerRequest = 3
```

Comment:
As I understand this, this variable controls the number of consecutive successful requests we require for a test to be considered successful. If that's a correct interpretation, the current name doesn't clearly communicate that to me. Maybe something like `requiredConsecutiveSuccesses` or similar would be clearer. It's hard to come up with a perfectly descriptive name for either of these vars, so it may also make sense to add comments describing the purpose of each of them.

Reply:
Renamed + added comments